app/src/main/java/no/usn/mob3000/domain/model/social/FriendData.kt

```diff
@@ -0,0 +1,14 @@
+package no.usn.mob3000.domain.model.social
+/**
+* @author Hussein Abdul-Ameer
+* created on 05.11.2024
```

Comment:
You are not using a docstring parameter for creation date. Additionally, creation date must conform with ISO8601.